docs/general-usage/graphql-types.rst

```diff
@@ -142,13 +142,29 @@ the ``ImageRenditionObjectType`` provides the following fields:
 ::
 
     id: ID
-    filterSpec: String!
+    url: String
     file: String!
     width: Int
     height: Int
-    focalPointKey: String!
+    aspectRatio: Float!
+    sizes: String!
     image: ImageObjectType!
-    url: String
```

Comment:
question: I don’t think I understand those changes – is this just unrelated cleanup?

Reply:
it is unrelated cleanup. noticed this while reading the docs. Was in the context and made sense to do it